### PR TITLE
feat: video hero overlay

### PR DESCRIPTION
**Pull request by GeronimoJohn**
# Description

Created the VideoHeroOverlay Component for our Single Video hero. 

[- Link to Basecamp Todo
](https://3.basecamp.com/3105655/buckets/30300130/todos/5566230905)

# How should this PR be QA Tested?
**Requirements:** 
Visit the [watch page](https://watch-git-22-08-jg-fix-video-overlay-jesusfilm.vercel.app/)
Visit a content video

- [x] it should render the Video Image with gradient as a background
- [x] it should render the Hero Texture on top of the video image
- [x] it should render the Video Title
- [x] it should render the duration of the video
- [x] it should render the Play Video Button
- [x] it should render the Mute Icon Button
- [x] it should play the video on Play Video Button click
- [x] it should close the overlay on Play Video Button click
- [x] it should play the video on Mute Icon click
- [x] it should close the overlay on Mute Icon click

# Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged into main

### CODE REVIEW

**Review by jernst7 — COMMENTED** [review]
Looks good to me

File: apps/watch/src/components/Hero/VideoHero/VideoHeroOverlay/VideoHeroOverlay.tsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        layout="fill"
+        objectFit="cover"
+        style={{
+          zIndex: 3
+        }}
+      />
+      <Container maxWidth="xl" sx={{ zIndex: 4 }}>
+        <Stack spacing={{ xs: 2, lg: 5 }}>
+          <IconButton
+            onClick={handleMute}
```
Comment (storyworks) [inline apps/watch/src/components/Hero/VideoHero/VideoHeroOverlay/VideoHeroOverlay.tsx:72]: Check with Lyuba if should `handlePlay`

File: apps/watch/src/components/VideoContentPage/VideoHero/VideoHeroOverlay/VideoHeroOverlay.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
+import { Meta, Story } from '@storybook/react'
```
Comment (tataihono) [inline apps/watch/src/components/VideoContentPage/VideoHero/VideoHeroOverlay/VideoHeroOverlay.stories.tsx:1]: remove this file. capture in VideoHero.

**Review by tataihono — CHANGES_REQUESTED** [review]
looks good just remove a story.

**Review by tataihono — APPROVED** [review]

**Review by tataihono — APPROVED** [review]

**Pull request merged**